rel/overlay/etc/default.ini

```diff
@@ -468,6 +476,22 @@ hash_algorithms = sha256, sha
 ;update_db = true
 
 ;[view_updater]
+; couch_work_queue:
```

Comment:
Hmm, maybe we shouldn't mention `couch_work_queue` per-se, that's seems too low level of a detail. It may  be better to keep it general.

> Configure queue capacity used during indexing. These apply to both the queue between the changes feed and 
the JS mapper, and between the JS mapper and the disk writer. Whichever limit happens to be hit first is the one that takes effect